src/Symfony/Component/DependencyInjection/Loader/Config/ServicesConfig.php

```diff
@@ -111,32 +104,19 @@
  *   public?: bool,
  *   deprecated?: Deprecation,
  * }
- * @psalm-type Services = array<string, Definition|Alias|Prototype|Stack>|array<class-string, Arguments|null>
+ * @psalm-type Services = array{
+ *   _defaults: Defaults,
+ *   _instanceof: Instanceof,
+ *   ...<string, Definition|Alias|Prototype|Stack>|array<class-string, Arguments|null>
```

Comment:
Ideally that would be like this as you can mix class-string and regular service names as array key.
```
 *   ...<class-string, Arguments|null>
 *   ...<string, Definition|Alias|Prototype|Stack>
```

For now, we need to define a more relaxed type.
```suggestion
 *   ...<string, Definition|Alias|Prototype|Stack>|Arguments|null>
```

Reply:
thanks, all fixed